src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -585,6 +592,15 @@ public Builder appCommandOutputFormat(String appCommandOutputFormat) {
     }
 
     /**
+     * Sets structured json logs for the stderr output. Supported values include 'never' (default),
+     * 'always', 'terminal', etc.
+     */
+    public Builder appCommandShowStructuredLogs(String appCommandShowStructuredLogs) {
```

Comment:
the `appCommand` prefix, while ugly, is to show clearly that the scope of the parameter is for gcloud command execution (e.g. `CloudSdk#runAppCommand` as opposed to a dev server command or appcfg etc.

I'm not denying though that this whole thing could be refactored to avoid this, but I don't think this PR is the place for it